### PR TITLE
Create script that allows JupiterOne integration instance creation for

### PR DESCRIPTION
**Pull request by austinkelleher**
every Google Cloud project in an organization

### CODE REVIEW

File: docs/jupiterone.md [hunk 1]
```diff
@@ -91,6 +91,84 @@ NOTE: You may also create a service account key using the
...
+Commands:
+  []  Default command: Run the organization setup
+
+For more info, run any command with the `--help` flag:
+  $ JupiterOne Google Cloud Organization Integration Setup --help
+
+Options:
+  --jupiterone-account-id <jupiteroneAccountId>  (Required) JupiterOne Account ID
+  --jupiterone-api-key <jupiteroneApiKey>        (Required) JupiterOne API Key
+  --google-access-token <googleAccessToken>      (Required) JupiterOne API Key
```
Comment (ndowmon) [inline docs/jupiterone.md:128]:
```suggestion
  --google-access-token <googleAccessToken>      (Required) Google Cloud Access Token
```

File: docs/jupiterone.md [hunk 2]
```diff
@@ -91,6 +91,84 @@ NOTE: You may also create a service account key using the
...
+  --project-id [projectId]                       (Optional) Array of project IDs to create integration instances with
```
Comment (ndowmon) [inline docs/jupiterone.md:129]:
```suggestion
  --project-id [projectId]                       (Optional) Project ID to create integration instances with. Passing multiple `--project-id` flags allows for ingestion of multiple projects.
```

File: scripts/organizationSetup.test.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        recordFailedRequests: true,
+      },
+    });
+  });
+
+  afterEach(async () => {
+    await recording.stop();
+  });
+
+  test('should iterate only specific projects when projectIds supplied', async () => {
```
Comment (ndowmon) [inline scripts/organizationSetup.test.ts:56]: No `expect`s here... is it worth adding? or is the resolved promise enough to know that this works?

**Review by ndowmon — APPROVED** [review]
Thanks! Approved. A few small questions.

File: scripts/organizationSetup.test.ts (hunk 3 above)
Comment (austinkelleher) [reply scripts/organizationSetup.test.ts:56]: I agree. I do want to add some, but I was trying to get this out quick. I just wanted to validate myself using the recordings. Plan to add some assertions later.

**Review by ndowmon — APPROVED** [review]

File: docs/jupiterone.md [hunk 4]
```diff
@@ -91,6 +91,84 @@ NOTE: You may also create a service account key using the
...
+
+#### Running
+
+The following shows all of the options that are exposed by the CLI.
+
+```
+JupiterOne Google Cloud Organization Integration Setup
+
+Usage:
+  $ JupiterOne Google Cloud Organization Integration Setup []
```
Comment (aiwilliams) [inline docs/jupiterone.md:117]: Wait, wut? I must be misreading this. 😆 Wouldn't it be better as `$ jupiterone-organization-setup`?

**Pull request merged**